RELEASES.md

```diff
@@ -1,5 +1,27 @@
 # Releases
 
+## 2.5.11 (January 15th, 2021)
+
+### Changelog
+
+- #1452 [BUG] v2.5.10 Snippet Initialization fails to redirect proxied functions -- causing terminal exception
+- #1434 Adding cookie path to make AI client SDK work behind Azure Application Gateway
```

Comment:
This is not in this release

Reply:
If you look at the milestone it has all of the changes attached (I did that eariler in the week)
https://github.com/microsoft/ApplicationInsights-JS/milestone/46 and
https://github.com/microsoft/ApplicationInsights-JS/milestone/46?closed=1

Reply:
Oh, I see #1434 was actually abandoned and not included :-)

Reply:
oh my bad, I didnt check the comments of that PR. Thank you